### ISSUE TITLE
Does the program not accept files with a URL list?

### ISSUE THREAD

**Issue by themaneiac** [RipMeApp/ripme#1037]
* Ripme version: v1.7.69
* Java version: 1.8.0_191
* Operating system: Windows 10 Pro Build 17134

I can't find anything anywhere on how to make a file full of URLs that RipMe accepts.
I tried making a text file with a URL on each line but it didn't even show up in the file picker. When I typed in the name of the text file and pressed open, RipMe just hanged endlessly. I tried a file with 10 URLs and nothing seemed to happen. A friend tried looking in the source code for an answer but didn't find one. I'm assuming this is a bug. The file I have has 1829 URLs (and I'll have a second file with lots of URLs soon) so it's impractical to queue them one by one. I can confirm the links do work individually.
Thanks :)

**Comment by GenuineSounds** [comment]
I can confirm that it will definitely work for a list of URLs that are albums, pages, and other URLs that crawl.

Like (NSFW):
> https://www.reddit.com/r/LipsThatGrip/
> https://www.reddit.com/r/nsfw/
> https://www.reddit.com/r/nsfw2/
> Etc...

I assume you are actually trying to download a list of individual endpoint files instead of pages.

That might help the devs figure it out if that information is correct.

**Comment by themaneiac** [comment]
They are all like this:

http://0bsydia.deviantart.com/gallery/?catpath=/
http://0ron0.deviantart.com/gallery/?catpath=/
http://0sweetcoconut0.deviantart.com/gallery/?catpath=/
http://11-shadow.deviantart.com/gallery/?catpath=/
http://1420517047.deviantart.com/gallery/?catpath=/
http://17paintcolor.deviantart.com/gallery/?catpath=/
http://1992zepeda.deviantart.com/gallery/?catpath=/
http://1an1.deviantart.com/gallery/?catpath=/
http://1deathpony1.deviantart.com/gallery/?catpath=/
http://1jaz.deviantart.com/gallery/?catpath=/

**Comment by themaneiac** [comment]
Just to clarify, I'm talking about this button 
![image](https://user-images.githubusercontent.com/6430050/47952089-7be31b80-df27-11e8-91aa-29d1d6b2b99d.png)

**Comment by GenuineSounds** [comment]
Removing "?catpath=/" from the URLs will fox that problem.

**Comment by GenuineSounds** [comment]
And just for clarification, this is a workaround not a fix. This is just to get you up and running.

**Comment by themaneiac** [comment]
Ok, I figured it out.. Both of us were wrong. "?catpath=/" is required to download all the images somebody made. When I was downloading profiles one by one I copied the URL from the browser and added "?catpath=/" to the end of it. What I didn't realize is the huge list of URLs I got were different.
URLs like this work:
https://www.deviantart.com/luminaura/gallery/?catpath=/
URLs like this don't:
http://luminaura.deviantart.com/gallery/?catpath=/

An actual bug (or something else interfering) here is the file open window doesn't show _anything_, no matter what I click in it
![image](https://user-images.githubusercontent.com/6430050/47952852-dcc42100-df32-11e8-865a-c1241070315b.png)
